tests/modules/annotations/resources/test_create_annotation.py

```diff
@@ -46,14 +45,18 @@ def test_create_failures(
     )
     assert response.json['message'] == 'bounds value is invalid'
 
+    with mock.patch('app.modules.asset_groups.tasks.delete_remote'):
```

Comment:
I think this would be cleaner as a clone.delete_all()
that would handle this mocking in one place rather than every call

Reply:
`clone.cleanup()` sounds like a good idea!

> I set it up so that the Asset group, once cloned, wasn't deleted.

Does it mean you've done this somewhere?

Reply:
Previously I'd removed the calls to clone.cleanup. This meant that the asset groups, these special ones, once cloned would hang about to prevent them having to be recloned. 
I just read through my comment above again and it really wasn't clear. This is what happens when you change your mind and edit a comment as opposed to deleteing it and starting again. Apologies.